src/components/ProductBlock/index.tsx

```
@@ -0,0 +1,83 @@
+    url: string,
+  }
+}): JSX.Element {
+  const location = useLocation();
+  const f_class = clsx(className, styles.indigo ? styles.productCardIndigo : styles.productCardTeal, light && styles.light);
+  return (
+    link
+      ? <Link {...rest} className={f_class} to={`${link}${location.search}`}>
+        {heading}
+        <div
+          className={styles.body}
+          dangerouslySetInnerHTML={{ __html: body }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)